code/game/machinery/computer/buildandrepair.dm

```diff
@@ -248,6 +248,13 @@
 	name = "circuit board (Library Visitor Console)"
 	build_path = /obj/machinery/computer/libraryconsole
 	origin_tech = "programming=1"
+/obj/item/weapon/circuitboard/bot_upgrade_boost
```

Comment:
Umm. By making it a circuitboard you may run into weird issues,such as this being able to be used to build a computer,which will cause weird shit. Maybe it'd be better to just define its own type under weapon/